TODO.txt

```diff
@@ -17,10 +16,14 @@ Version 0.14
   deprecation warning test for this alias.
 * Remove deprecated ``sigma_range`` kwargs in ``skimage.restoration.denoise_bilateral``
   and corresponding tests.
+* Remove deprecated `normalise` and corresponding test for ``skimage.feature.hog``. 
```

Comment:
@stefanv the deprecation had started in January with https://github.com/scikit-image/scikit-image/pull/1899 (0.12.3 was tagged in April). So, 0.13 will be the last release with that warning (actually, error), and 0.14 will have this kw removed.
I guess I'll just change the message to be more definite about the end of deprecation.